chart/k8gb/templates/operator.yaml

```diff
@@ -86,3 +86,7 @@ spec:
             - name: ROUTE53_ENABLED
               value: "true"
             {{ end }}
+            {{ if .Values.ns1.enabled }}
+            - name: NS1_ENABLED
+              value: "true"
```

Comment:
does helm support something like this ? 
```yaml
 - name: ROUTE53_ENABLED
    value:  {{ .Values.route53.enabled }}
 - name: NS1_ENABLED
    value:  {{ .Values.ns1.enabled }}

```

Reply:
@kuritka maybe but it will be declared but empty variables, do not think we want to pollute the environment with them